gdsfactory/component.py

```diff
@@ -736,14 +736,14 @@ def plot_netlist(
         )
         return G
 
-    def over_under(self, layer: LayerSpec, distance: float = 1.0) -> None:
+    def over_under(self, layer: LayerSpec, distance: int = 1) -> None:
         """Flattens and performs over-under on a layer in the Component.
```

Comment:
Do you really need to flatten? I think this makes it strictly bigger, so theoretically it shouldn't be necessary to flatten. But maybe a flag? Otherwise, sure flattening works.